app/models/ext_management_system.rb

```diff
@@ -292,6 +292,10 @@ def supports_authentication?(authtype)
     authtype.to_s == "default"
   end
 
+  def supports_metrics?
```

Comment:
TODO this should be a supports_feature_mixin but kubernets already has this method defined